<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyMessenger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonAddContact">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAddContact">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChatSend">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxChatName">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>381</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Чат не выбран</string>
    </property>
    <widget class="QPushButton" name="pushButtonFormatB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../img/icon/b.jpg</normaloff>../../img/icon/b.jpg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFormatI">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../img/icon/i.jpg</normaloff>../../img/icon/i.jpg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFormatU">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>280</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../img/icon/u.jpg</normaloff>../../img/icon/u.jpg</iconset>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditChatInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditChat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>361</width>
       <height>241</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSmile">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/smile/ab.gif</normaloff>../img/smile/ab.gif</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="labelSmileBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>280</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../img/smile/ab.gif</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidgetContactList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <zorder>pushButtonAddContact</zorder>
   <zorder>lineEditAddContact</zorder>
   <zorder>groupBoxChatName</zorder>
   <zorder>pushButtonChatSend</zorder>
   <zorder>listWidgetContactList</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="quit"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="about"/>
   </widget>
   <widget class="QMenu" name="settings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="profile"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="settings"/>
   <addaction name="help"/>
  </widget>
  <action name="quit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="profile">
   <property name="text">
    <string>Профиль</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
